src/react-pure-datepicker.js

```diff
@@ -294,12 +294,15 @@ class PureDatepicker extends React.Component {
             <div className="calendarWrap">
               <div className="weekdays-names">
                 {
-                  weekDaysNames.map(weekDayName => (
-                    <div
-                      key={weekDayName}
-                      className={`${weekDayName.toLowerCase().startsWith('s') ? 'weekend' : ''} weekDayNameShort`}
-                    >{weekDayName}</div>
-                  ))
+                  weekDaysNames.map(weekDayName => {
+                    const index = weekDaysNamesShort.indexOf(weekDayName);
```

Comment:
Will be better if you will add  key value map with weekday type. IndexOf on every iteration it's not a good idea